.github/workflows/pages.yml

```diff
@@ -1,7 +1,6 @@
 name: pages
 on:
   push:
-    branches: [ main ]
```

Comment:
I'd keep it

Reply:
I think it must have been changed for debugging purposes. I've reverted the change